modules/sdl/2.0.8/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--cxxopt=-std=c++17'
```

Comment:
this would probably be better added in the `copts` of the `cc_library` instead of as a build flag in the presubmit.